src/huggingface/configs/config.jl

```diff
@@ -0,0 +1,30 @@
+using JSON
```

Comment:
JSON3 has great support for de/serializing AbstractDict, see: https://quinnj.github.io/JSON3.jl/stable/#StructTypes.DictType-1 . (No need to change here, just a gentle reminder.😃 )

Reply:
What's the different between using JSON and JSON3 ?

Reply:
Since that AbstractHGFConfig is already a subtype of AbstractDict, I guess with JSON3, https://github.com/chengchingwen/Transformers.jl/pull/9/files/dac873007f6289b7e0f74f6a70ac22f3905c43ff#diff-56f47ab93bc96fbe6afa99bce9821a9aR11-R28 this part can be slightly simplified.